scripts/Jenkinsfile

```diff
@@ -45,6 +45,41 @@ pipeline
                 }
             }
         }
+
+
+        stage('Install vLLM')
+        {
+            steps
+            {
+                sh '''
+                    . preflight_qeff/bin/activate
+                    git clone https://github.com/vllm-project/vllm.git
+                    cd vllm
+                    git checkout v0.6.0
+                    git apply /opt/qti-aic/integrations/vllm/qaic_vllm.patch
+                    export VLLM_TARGET_DEVICE="qaic"
+                    pip install -e .
+                '''
+            }
+        }
+
+
+        stage('vLLM Test')
+            {
+                steps
+                {
+                    
+            timeout(time: 660, unit: 'MINUTES') {
+                    sh '''
+                    . preflight_qeff/bin/activate
+                    pytest --disable-warnings -s -v tests/vllm --junitxml=tests/tests_log3.xml
+                    junitparser merge tests/tests_log1.xml tests/tests_log2.xml tests/tests_log.xml tests/tests_log3.xml
```

Comment:
please rebase, the jenkinsfile already had some change after PR #116  merge.
Also you need to do `junitparser merge split_1.xml split_2.xml split_3.xml split_4.xml tests/tests_log.xml`

`tests_log.xml` should be the output file and at the end of the command, as this will be used by junit to display test results at the end of the pipeline in `Post`

Reply:
Done.